config/src/config/execution_config.rs

```diff
@@ -51,7 +51,7 @@ impl Default for ExecutionConfig {
             num_proof_reading_threads: 32,
             paranoid_type_verification: true,
             paranoid_hot_potato_verification: true,
-            processed_transactions_detailed_counters: false,
+            processed_transactions_detailed_counters: true,
```

Comment:
this cannot default to true, as it would create too many counters on too many nodes.

Reply:
Yes, enabled it for temporary debugging. Will revert.